x-pack/plugins/apm/server/routes/metrics/by_agent/serverless/cold_start_count.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { termQuery } from '@kbn/observability-plugin/server';
+import {
+  FAAS_COLDSTART,
+  FAAS_ID,
+} from '../../../../../common/elasticsearch_fieldnames';
+import { Setup } from '../../../../lib/helpers/setup_request';
+import { fetchAndTransformMetrics } from '../../fetch_and_transform_metrics';
+import { ChartBase } from '../../types';
+
+const chartBase: ChartBase = {
+  title: i18n.translate('xpack.apm.agentMetrics.serverless.coldStart', {
+    defaultMessage: 'Cold start',
+  }),
+  key: 'cold_start_count',
+  type: 'linemark',
+  yUnit: 'number',
+  series: {
+    coldStart: {
+      title: i18n.translate('xpack.apm.agentMetrics.serverless.coldStart', {
+        defaultMessage: 'Cold start',
+      }),
+    },
+  },
+};
+
+export function getColdStartCount({
+  environment,
+  kuery,
+  setup,
+  serviceName,
+  faasId,
+  start,
+  end,
+}: {
+  environment: string;
+  kuery: string;
+  setup: Setup;
+  serviceName: string;
+  faasId?: string;
+  start: number;
+  end: number;
+}) {
+  return fetchAndTransformMetrics({
+    environment,
+    kuery,
+    setup,
+    serviceName,
+    start,
+    end,
+    chartBase,
+    aggs: { coldStart: { sum: { field: FAAS_COLDSTART } } },
+    additionalFilters: [
+      { term: { [FAAS_COLDSTART]: true } },
+      ...termQuery(FAAS_ID, faasId),
```

Comment:
any reason why one uses `termQuery` and the other one does not?

Reply:
no reason. changing it